File: extractor-core/src/main/java/ru/mydesignstudio/database/metadata/extractor/extractors/model/ArgumentModel.java
package ru.mydesignstudio.database.metadata.extractor.extractors.model;

import lombok.Data;

@Data
public class ArgumentModel {
  private String schemaName;
  private String procedureName;
  private String argumentName;
  private String inOut;
  private String dataType;
  private Integer dataLength;
  private Integer dataPrecision;
  private Integer dataScale;
  private String defaulted;
  private String defaultValue;
}

File: extractor-core/src/main/kotlin/ru/mydesignstudio/database/metadata/extractor/extract/result/ArgumentModel.kt
package ru.mydesignstudio.database.metadata.extractor.extract.result

data class ArgumentModel(
        val schemaName: String,
        val procedureName: String,
        val argumentName: String,
        val inOut: String,
        val dataType: String,
        val dataLength: Int,
        val dataPrecision: Int,
        val dataScale: Int,
        val defaulted: String,
        val defaultValue: String
)